data/hai/hai reveal 6 end.txt

```diff
@@ -109,15 +117,23 @@ mission "Hai Reveal [C02] Cleanup"
 			choice
 				`	"The Hai are no longer a secret in human space, but they are concerned how you ended up here."`
 					goto revealed
+					to display
+						not "Hai Reveal [C02] Cleanup: label revealed"
 				`	"What happened to the ship that brought you here?"`
 					goto kidnappers
+					to display
+						not "Hai Reveal [C02] Cleanup: label kidnappers"
 				`	"Well, lets get you home before anything else happens."`
 					goto depart
 			label revealed
+			action
+				set "Hai Reveal [C02] Cleanup: label revealed"
 			`	"Ah, we thought we saw something about that, but our comms are spotty, so we didn't rely on it. We all actually signed up to go to Mountaintop to take part in the facade, so it seemed especially suspicious."`
 			`	He shrugs. "An abundance of caution seemed warranted; we were kidnapped, after all."`
 				goto responses
 			label kidnappers
+			action
+				set "Hai Reveal [C02] Cleanup: label kidnappers"
 			`	"We were en route to somewhere after the ship left Hai space. We don't really know what happened, but they received some kind of message that left them very concerned. They said something about hiding somewhere safe and landed here."`
 			`	At this point, he frowns in the way Hai do. "We were set down here for days and days. Our kidnappers couldn't seem to work out what they should do. Eventually, a woman landed next to us in this ship here," he explains, gesturing at the Star Barge. "She had them ferry us and all our stuff out to this ship, and then she took charge and flew off with them. That was when we sent the message. We were partly afraid it was some kind of elaborate trap when we saw that we could communicate to a Hai address directly. We never used to be able to communicate between human space and ours like that. I guess we were lucky this time."`
```

Comment:
How do the Hai frown in the way they do?

Reply:
I just removed the redundant information that the Hai is frowning in the way that Hai frown, since we know Hai always frown in the way that Hai frown.